Workbooks/ThycoticWorkbook.json

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
Change workbooksMetadata.json file, so that it will contain a new section for Thycotic connector workbook. Please refer step 2 of https://github.com/Azure/Azure-Sentinel/blob/30b5ade4aa74942aca543bbb33db2d360e08b810/Workbooks/README.md for more details. You can check the metadata file https://github.com/Azure/Azure-Sentinel/blob/master/Workbooks/WorkbooksMetadata.json with metadata of other connectors sections as reference.